api/users/views.py

```diff
@@ -488,6 +494,9 @@ class UserEmploymentList(JSONAPIBaseView, generics.ListCreateAPIView, UserMixin,
         ReadOnlyOrCurrentUserProfile,
     )
 
+    required_read_scopes = [CoreScopes.USERS_READ, CoreScopes.EMPLOYMENT_READ]
+    required_write_scopes = [CoreScopes.NULL]
```

Comment:
I believe this should be CoreScopes.USERS_WRITE because this is class-based view inherits from the ListCreateAPIView and is used to create a new UserEmployment object.